omod/src/main/resources/config.xml

```diff
@@ -46,5 +46,10 @@
 		<file>messages_es.properties</file>
 	</messages>
 	<!-- /Internationalization -->
+
+	<advice>
+		<point>org.openmrs.api.PatientService</point>
```

Comment:
Shouldn't this be on PersonService?

Reply:
Yes correct, Actually I plan to change it to PersonService while implementing the user location management(To address Patients and Users). I addressed this PR as only for Patients, So I handle only with high-level PatientService. 

I will change it to Persons and will update the PR content.

Reply:
I thought PersonService would automatically deal with patients because they are persons. Not so?

Reply:
Nope, AFAIK We need to access this PatientService